contributors/devel/godep.md

```diff
@@ -15,6 +15,17 @@ the tools.
 
 This doc will focus on predictability and reproducibility.
 
+## Justifications for an update
+
+Before you update a dependency, take a moment to consider why it should be 
+updated. Valid reasons include:
+ 1. We need new functionality that is in a later version.
+ 2. New APIs in the dependency are better than the old ones.
+ 3. Bugs that matter to Kubernetes have been fixed.
+ 4. We need dependency A and there is a transitive dependency B.
+ 5. Kubernetes has an older level of a dependency that is precluding being able
+to work with other projects in the ecosystem.
+
 ## Theory of operation
```

Comment:
Here's a list from @thockin (over email)

- Bugs were fixed (that impact us)
- Security issues were fixed (even if they don't impact us yet)
- Perf /scale / efficiency improvements
- Testing improvements
- Needed transitively by other things we need
- Our lock on old revs is causing friction in consumers of k8s (e.g. API evolved, but we are old)

Reply:
> Here's a list from @thockin (over email)

Our lists were close.  I do find the testing improvements one a little bit weak since we don't actually run upstream tests, but I suppose that I can buy the argument that they probably also fixed bugs.